extensions/cocostudio/timeline/CCBoneNode.js

```diff
@@ -279,8 +284,8 @@ ccs.BoneNode = (function () {
 
         setLocalZOrder: function (localZOrder) {
             Node.prototype.setLocalZOrder.call(this, localZOrder);
-            if (this._rootSkeleton != null)
-                this._rootSkeleton._sortedAllBonesDirty = true;
+            //if (this._rootSkeleton != null)
```

Comment:
need dirty the bones' sequence  `SkeletonNode::_subOrderedAllBones` for draw skins as zorder